packages/zowe-explorer/src/dataset/DatasetTree.ts

```diff
@@ -1151,8 +1152,15 @@ export class DatasetTree extends ZoweTreeProvider implements IZoweTree<IZoweData
     private async renameDataSetMember(node: IZoweDatasetTreeNode) {
         const beforeMemberName = node.label as string;
         const dataSetName = node.getParent().getLabel() as string;
-        const options: vscode.InputBoxOptions = { value: beforeMemberName };
-        let afterMemberName = await UIViews.inputBox(options);
+        const options: vscode.InputBoxOptions = {
+            value: beforeMemberName,
+            validateInput: (text) => {
+                return validateDataSetName(text) === true
+                    ? null
+                    : localize("dataset.validation", "Enter valid dataset name");
```

Comment:
Should this say "Enter valid member name" instead, and perform different validation? For example, `IBMUSER.TEST` is a valid data set name but not a valid member name.

Reply:
addressed in commit [44418ad](https://github.com/zowe/vscode-extension-for-zowe/pull/1977/commits/44418add0d80197dbffdd8bd4f55121ae869e315)